python/tvm/relay/frontend/paddlepaddle.py

```diff
@@ -475,6 +499,39 @@ def convert_elementwise_op(g, op, block):
     g.add_node(op.output("Out")[0], out)
 
 
+def convert_linspace(g, op, block):
+    """Operator converter for linspace."""
+
+    start = g.get_node(op.input("Start")[0])
+    stop = g.get_node(op.input("Stop")[0])
+    num = g.get_node(op.input("Num")[0])
+    dtype = _convert_dtype_value(op.attr("dtype"))
+
+    start = _op.cast(start, dtype)
+    stop = _op.cast(stop, dtype)
+    num = _op.cast(num, dtype)
+
+    if dtype in ["int32", "float32"]:
+        tmp_dtype = "float32"
+    else:
+        tmp_dtype = "float64"
+    start = _op.cast(start, tmp_dtype)
```

Comment:
There should be two cases where you can infer and two cases where you can't infer. 
If it is a static shape, the op to TVM is very small;
If the value cannot be inferred from the dynamic shape, more op will be converted to TVM, which may affect performance.

Reply:
if the start,stop and num value can be inferred, that's what you implemented before.
There is no need to add op like where, subtract, divide, etc

Reply:
Yes, but I think tvm will optimize automatically. If the `start` `stop` and `num` value can be inferred, these ops will be eliminated by FoldConstant pass in relay, it only outputs a relay.Constant.  https://github.com/apache/tvm/blob/main/python/tvm/relay/frontend/paddlepaddle.py#L2572

Reply:
I tested the case where the values can be infered, the relay IR as follow.
```
def @main() {
  meta[relay.Constant][0]
}
```

Reply:
👌